test/e2e-go/features/transactions/asset_test.go

```diff
@@ -539,6 +539,6 @@ func TestAssetSend(t *testing.T) {
 	_, err = helperFillSignBroadcast(client, wh, extra, tx, err)
 	a.NoError(err)
 
-	_, err = client.SendPaymentFromWallet(wh, nil, extra, "", 0, 0, nil, account0, 0, 0)
+	_, err = client.SendPaymentFromWallet(wh, nil, extra, "", 0, 0, nil, account0, [32]byte{}, 0, 0)
```

Comment:
The fact that a caller that doesn't care about the `lease` has to supply `[32]byte{}` rather than `nil` seems to break existing patterns to me, but not a big deal.

Reply:
I think that we should be able to pass nil into a []byte{} array. not so sure about a [32]byte{} array.

Reply:
(i.e. it would work for slice, but I'm not sure about an array )

Reply:
Sounds good--are you thinking it's better to have `SendPaymentFromWallet` and `SendPaymentFromWalletWithLease`?

Reply:
Just read your top-level comment--sounds good.